src/Gui/Quarter/QuarterWidget.cpp

```diff
@@ -954,6 +953,19 @@ bool QuarterWidget::viewportEvent(QEvent* event)
             return false;
         }
     }
+    else if (event->type() == QEvent::Wheel) {
+        auto wheel = static_cast<QWheelEvent*>(event);
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
```

Comment:
I think this check should actually be for Qt 5.14, when the `position()` function was added.

Reply:
Good catch! I just noticed there are pieces [like this](https://github.com/FreeCAD/FreeCAD/blob/07a6c62aef90f8544327596cc41175fc6b4cd74a/src/Gui/Quarter/Mouse.cpp#L163), but there are also snippets [like this](https://github.com/FreeCAD/FreeCAD/blob/07a6c62aef90f8544327596cc41175fc6b4cd74a/src/Gui/ListWidgetDragBugFix.cpp#L44-L48) (which I noticed and took inspiration from, to be honest :sweat_smile:). Do you think that an issue should be open in order to fix analogous cases?

Reply:
This one caught my eye because my linter complained about it -- if you'd like to fix the others you are welcome to, but I wouldn't describe it as a "high priority item" 😀 .

Reply:
Ok, forget what I said :sweat_smile:: the other occurrences of the version check in correspondence to the usage of `pos()` instead of `position` are on `QMouseEvent` or other things that seem to have a different deprecation timeframe (the code looks fine for them).

In any case I changed the check as you suggested, which is also confirmed by `QT_DEPRECATED_VERSION_X_5_15` in `qevent.h`. Thank you! :blush: